docs/io/fits/appendix/faq.rst

```diff
@@ -493,7 +488,7 @@ Comparison with Other FITS Readers
 What is the difference between astropy.io.fits and fitsio?
 ----------------------------------------------------------
 
-The `astropy.io.fits` module (originally PyFITS) is a "pure Python" FITS
```

Comment:
I think it is fine to leave this in as a "history lesson".

Reply:
No strong feeling but the section about difference between `astropy.io.fits` and `pyfits` should cover that already?!

Reply:
It is also frequently asked whether PyFITS is the same thing as `astropy.io.fits`, so does not hurt to be a bit repetitive in FAQ? 😅

Reply:
If you insist I'll re-add it. My intention was to remove it wherever possible. 😄

Reply:
Let's see what @saimn and @drdavella think about this. They are the other deputies, after all.

Reply:
I'll add it again.